tutorials/bios-boot-tutorial/bios-boot-tutorial.py

```diff
@@ -103,6 +103,7 @@ def startNode(nodeIndex, account):
     cmd = (
         args.nodeos +
         '    --max-irreversible-block-age -1'
+        '    --max-transaction-time=1000'
```

Comment:
max-transaction-time should never be set larger than block time.

Reply:
This was taken from Release-2.0.x. I could not find default block time. Should it be taken out?

Reply:
That should be removed from release-2.0.x then as well. The default of 30 is likely too low hence the argument. I would recommend setting it larger than 30 but less than block time.

Reply:
Do you know how much block time is? I cannot find where it is defined.

Reply:
Thanks! Was searching for `block_time`. Will set `max-transaction-time` to 200 to be on safe side.